downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -103,8 +103,73 @@ To view other bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https
 
 ### New features
 
+#### New Runtime Java APIs
+
+- To clone an `anydata` value with another subtype of `anydata` type, the following API is introduced in the `io.ballerina.runtime.api.utils.ValueUtils` class.
+    ```java
+    public static Object convert(Object value, Type targetType);
+    ```
+- `getFunctionName()` and `getPathParameters()` APIs are introduced in the runtime `io.ballerina.runtime.api.Environment` class. They provide the Ballerina function name and path parameters associated with an external Java method, respectively.
+
 ### Improvements
 
+#### Support for command line arguments of built-in subtypes
+
+The command-line arguments of Ballerina built-in subtypes are now supported to be parsed as operands.
+
+For the following `main` function,
+```ballerina
+public function main(byte byteVal, string:Char charVal, int:Signed8 int8Val) {
+}
+```
+the values can be passed through the command line arguments as follows.
+```
+bal run -- 1 b 33
+```
+
+#### Support ambiguous union type configurable variables
+
+When a structural value is provided for a configurable variable of a union type that includes more than one type descriptor, the inherent type used will be the first (leftmost) type descriptor, from which the value can be constructed.
+
+For example, given the following Ballerina code
+```ballerina
+type Person record {|
+    string name;
+    string city;
+|};
+
+configurable map<string>|Person configVar = ?;
+```
+and the `Config.toml` file contents as follows,
+```toml
+configVar = {name = "Jack", city = "Colombo"}
+```
+`configVar` is instantiated with the inherent type `map<string>`.
+
+#### Support binding of resource functions to a generic native method
+
+A new way has been introduced to support the binding of any resource function to a generic native method, regardless of the resource path parameters. The generic native method should be defined with a `BArray` parameter, which represents all path parameters. To avoid errors due to overloaded methods, it is recommended to define parameter type constraints as well.
```

Comment:
```suggestion
A new way has been introduced to support the binding of any resource function to a generic native method regardless of the resource path parameters. The generic native method should be defined with a `BArray` parameter, which represents all the path parameters. To avoid errors due to overloaded methods, it is recommended to define the parameter type constraints as well.
```